### PR TITLE
Update labels.json

### PR DESCRIPTION
**Pull request by wargreen**
Update translations : avoid literal translation and some improvements for specifics terms.

### CODE REVIEW

**Comment by sadko4u** [comment]
Thanks for the changes.
Merged into **devel** branch: 8deb25de1a21722e4240ff6f62ed7fef754032aa

**Comment by sadko4u** [comment]
Discarded the merge.

Since I'm not good with french, @trebmuh has some moments to discuss about translations.
So I think we need to find the compromise first before merging changes into **devel** branch.

**Comment by wargreen** [comment]
Ok, no problem for discuss ! Where can i find his comments ?

**Comment by sadko4u** [comment]
> Ok, no problem for discuss ! Where can i find his comments ?

I suggest to perform the discussion here. Please wait for a while. According to the feedback, some changes are really good but some changes are not well.

**Comment by sadko4u** [comment]
@trebmuh how are you about the suggested changes?

**Comment by trebmuh** [comment]
Hi there. I'm back after having been AFK a few days.
I'll look at this probably tomorrow.

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:484]: @sadko4u : could you please explain us a bit more about what it is/does from your dev point of view? I'm not sure to be a big fan of this translation proposal.

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:490]: @sadko4u : _same than line 484_

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:494]: It sounds/looks very wrong to me.
Proposal: "_Direction latérale_".

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:503]: Good ones.

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:507]: I'm unsure about what this is/does, but this proposal sounds wrong to me.
@sadko4u : could you please enlighten us about what this is/does?

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:515]: "largeur" is a feminine noun, hence the adjective needs to match it.
Please revert.

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:524]: Same than line 515. "largeur" is a feminine noun, hence the adjective needs to match it.
Please revert.

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:546]: It could be "étalement" too. In fact, spread in the music contexte is often translated to "étalement".

@sadko4u : could you please tell us about what it does/is? If "spread" is thought as "spreading some strawberry jam on bread", then "étalement" is the way to go. If not, your explanation will help us to find the best translation match.

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:564]: OK

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:568]: OK

File: res/i18n/fr/labels.json
Comment (trebmuh) [inline res/i18n/fr/labels.json:615]: OK

**Review by trebmuh — CHANGES_REQUESTED** [review]
Here are my feedbacks.

Apologize for the delay, I'm currently moving from a place to another one and starting a new job so my time is quite limited.

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:13]: It's a part of standard yaw/pitch/roll angle set for the camera rotation in the 3D space:
http://wiki.ogre3d.org/Creating+a+simple+first-person+camera+system#What_about_yaw_pitch_and_roll_

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:27]: It's a part of standard yaw/pitch/roll angle set for the camera rotation in the 3D space:
http://wiki.ogre3d.org/Creating+a+simple+first-person+camera+system#What_about_yaw_pitch_and_roll_

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:55]: Hue means here the part of the HSL (Hue/Saturation/Lightness) color model:
https://en.wikipedia.org/wiki/HSL_and_HSV

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:546]: From the trigger documentation: 'Dynamics spread - this knobs allows to add some dynamics to output signal.'
So here 'Spread' means that there is some amount of randomness is added to the velocity of the output signal.

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:268]: If you see the '_' in the key, that means that the word should be shortened to the around the same number of symbols that is before this character. So maybe 'Courbure' can be shortened to 'Cour' here?

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:238]: This is a text for the graph presentation. In original it is 'DELAY LINE' and should match to the same phrase for other languages.

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:12]: Please note that if some keys are grouped under the common object, then this object means the context where these keys are used. So here the context is related to the 3D navigaion in the Room Builder plugin.

**Comment by sadko4u** [comment]
@wargreen could you please add changes in your branch according to remarks done by @trebmuh ?

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:238]: So I propose "Ligne de délais", what is commonly used.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:342]: I think that we need to get a (direct and visual) relation between the translation of "mute" and "muted", for get an easyer and faster usage of the GUI, so i think that "Rendre muet" is good.
But i'm OK, "Silencier" is a good academic word.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:397]: According to the manual, it add a little fade to avoid clicks when the daw (or the signal path) don't add it.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:367]: If it is many ovensampling mode (and it is a lot in LSP) it seem justified. But the sentence can be a little long....

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:494]: This parameter control the patern (8 or bidirectional) of the "side" channel of an Mid/Side signal.
So i think that we can't translate "direction" but "directivité" (for "pattern" from polar pattern). The "side" word is more difficult, but litteraly it's "cotés" (plurials : we have 2 sides in the channel), and i don't find better wihout make a full sentence. Maybe "Directivité des cotés" ?

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:421]: In the GUI, this word is related to another control (in an array), and it is not related to an actual sentence. I think that this  "T" is justified : it's another control.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:268]: Maybe "Crbr" ? "Cour" don't seem really understandable to me.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:351]: It seem to be for the color of an object, so "Teinte" seem good.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:385]: Maybe "Panoramique", but it's longer.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:473]: I have never heard "Surgain" in the tech vocabulary. In this context, this stage add level, what is simply a "gain" in french.
So for me : "keep it simple" and "less is more" if it don't can be ambiguous.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:484]: It's for a IR begining, and "ecretage" is "clipping" in this "saturation" meaning.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:524]: I've change this cause in my head : it's the width of the signal, and in french "signal" is a masculin word.

**Comment by wargreen** [comment]
@sadko4u i've commented the @trebmuh comments (sorry for the time, some work here). I wait for resolve all conversation before finish the modifications.

**Comment by sadko4u** [comment]
Hi folks! Have you solved all your translation conflicts? The pull request is waiting for merge already for a long amount of time.

**Comment by wargreen** [comment]
Hi !
I have no news from @trebmuh about theses comments...
Maybe now he have less of work ? Ping @trebmuh !

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:218]: ping @wargreen

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:236]: ping @wargreen

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:238]: ok

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:268]: Crbr looks good to me

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:367]: "Sur-échantillonneur" is good enough, match the English one, and as you say is shorter.
Please revert.

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:385]: Let's go for "Panoramique", if we need to shorten it, then we'll switch to "Panora.".

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:397]: @sadko4u do you agree with it being a "declicker" ? Or is it intended to have a wider use that this "declicking" scenario case?

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:421]: I disagree, there is a éQ," before it. French typo. rules are stricts about what can be capitalized or not.
In this situation, is must not be so.

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:427]: ping @wargreen

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:473]: if that would be only a "gain", then it could have been the same in English.
@sadko4u : can you please tell us if that's only a "gain stage" or if that's more than that?

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:484]: @sadko4u : can you please explain us what does this do exactly?

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:494]: "Directivité des cotés" : OK for this one @wargreen.

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:507]: @sadko4u : ping

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:515]: ping @wargreen

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:524]: This adjective is related to the noun "largeur", so it definitely need to match that, hence it's "largeur positive".
Ping @wargreen.

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:546]: Thank @sadko4u. I'm all good for either Propagation, Diffusion, or Étalement then.
My preference would be "étalement.
@wargreen: ping

**Comment by trebmuh** [comment]
@sadko4u and @wargreen : appologize for the delay, I moved to another place for a new job on those last months. I've made my comments now. Hope all can be merged in devel soon.

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:507]: It's https://fr.wikipedia.org/wiki/Rapport_cyclique

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:236]: It seem obvious that is a graph. On many plugins (free and proprietary) it's not specified "graph".
What do you think @sadko4u ?

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:421]: Since it's two controls, and two initials of the control's names, it should be capitals.

File: res/i18n/fr/labels.json
Comment (wargreen) [reply res/i18n/fr/labels.json:342]: So what do we do ? Academic way or simpler and understandable way ?
Maybe one word @sadko4u ?

Comment (sadko4u) [reply res/i18n/fr/labels.json:342]: I don't understand the french at all. All what i can do - just refer to the documentation which describes the behaviour of the corresponding control:

```
Muting - when enabled, turns off sample playback for selected instrument when the Channel Control MIDI message is received.
```
https://lsp-plug.in/?page=manuals&section=multisampler_x12

So please refer to the control in term `what it does` instead of `what it is` when translating.
All misunderstanding can be simply solved by just reading the corresponding manual page for the corresponding plugin.

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:421]: I would like to admit with @wargreen. Here we just have two separate names.

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:236]: I think it's not too monosemantic. I would like to confirm with main (english) translation.
When preparing 1.2.0, we can review the names and maybe throw-out the 'Graph' word.

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:421]: As already said above, it's not French to write it like that.
Please, revert.

File: res/i18n/fr/labels.json
Comment (trebmuh) [reply res/i18n/fr/labels.json:55]: Capitalization of "Bande" is wrong here. It must be "bande".

**Comment by trebmuh** [comment]
From my perspective, I made comments about wargreen's translation proposals.
If it is needed, I can make a PR myself too, just let me know then.

**Comment by sadko4u** [comment]
Also, short information about 1.2.0. Most of language bundles have been moved here:
https://github.com/lsp-plugins/lsp-plugins-shared/tree/devel/res/main/i18n/fr

Also plugins can provide their own language bundles, for example:
https://github.com/lsp-plugins/lsp-plugins-mb-gate/tree/devel/res/main/i18n/fr
https://github.com/lsp-plugins/lsp-plugins-mb-expander/tree/devel/res/main/i18n/fr

So it would be nice if you could review all bundles and update them for 1.2.0 release.

File: res/i18n/fr/labels.json
Comment (sadko4u) [reply res/i18n/fr/labels.json:236]: Please keep it for now.

**Comment by sadko4u** [comment]
Hi all!
Any movement?

**Comment by wargreen** [comment]
@sadko4u Hi ! I've commit with many changes from the comments. If it's OK for @trebmuh i think it can be ready.

**Comment by sadko4u** [comment]
Merged changes to the **devel** branch: 07c5af3a1b82d3d1b18369f3707c8358f1bb6c63
Please consider further pull requests applied to the **devel** branch.

**Comment by sadko4u** [comment]
Available in 1.1.31 release!

**Pull request closed without merging**
